### ISSUE TITLE
Cannot generate token or login to Tesla via app link

### ISSUE THREAD

**Issue by dgruhlke** [Kemmey/Kemmey-TeslaWatch-Public#339]
When clicking the link to login to Tesla, I get access denied.  Sam for the auth app for Tesla in generating a token.  I do not have a VPN on.

**Comment by dgruhlke** [comment]
![A53AD4CC-9B07-45E1-8A2E-2583BB323277](https://user-images.githubusercontent.com/83901615/117564745-2bf2eb80-b073-11eb-990b-b0c522c2b491.jpeg)

**Comment by Kemmey** [comment]
@dgruhlke Yeah, seems like Tesla made some authentication changes this morning - most 3rd party apps are broken currently :-(

Entire community is working on a fix!
